### PR TITLE
Add GSOC 2021 summary post.

### PR DESCRIPTION
**Pull request by hmacdope**
The summary post for GSOC 2021! 

Hopefully just in time for NumFocus newsletter.

### CODE REVIEW

File: _posts/2021-9-14-gsoc2021-final.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+
+<p>
+<img
+src="{{ site.baseurl }}{{ site.images }}/mdanalysis-gsoc.png"
+title="Google Summer of Code 2021" alt="Google Summer of Code 2021"
+style="float: right; height: 12em; " />
+</p>
+
+Another year, another successful [Google Summer of Code][gsoc]! We had
+[two fantastic projects][gsoc-projects] this year, including the first trials of our MDAKit model for building an ecosystem of analysis modules. Both projects were a resounding success and represent an exciting new direction for MDAnalysis.  None of this would have been possible without our wonderful students: [Estefania Barreto-Ojeda][project-estefania], and [Orion Cohen][project-orion].
```
Comment (orbeckst) [inline _posts/2021-9-14-gsoc2021-final.md:14]: add their GH handles somewhere

File: _posts/2021-9-14-gsoc2021-final.md [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+You can read more about their projects and packages in their blog posts:
+* Estefania: [Curvature for MDA Universes ]({{ site.baseurl }}{% post_url 2021-09-02-gsoc-final-report-estefania%})
+* Orion: [Solvation Analysis]({{ site.baseurl }}{% post_url 2021-09-02-gsoc-final-report-orion%})
+
+
+Finally, a big thank-you to our GSoC mentors: @fiona-naughton, @lilyminium, @orbeckst,
+@IAlibay, @richardjgowers and @hmacdope. Their dedication, openness and engagement was
+crucial to the success of our students. 
+
+— Org Admins (@richardjgowers, @IAlibay, @orbeckst)
```
Comment (orbeckst) [inline _posts/2021-9-14-gsoc2021-final.md:25]: Let's rewrite
```suggestion
We immensely enjoyed working with our GSoC students and look forward to seeing where they take their projects next.

— Mentors @hmacdope, @lilyminium, @richardjgowers, @fiona-naughton, @orbeckst, @IAlibay
```

Comment (orbeckst) [reply _posts/2021-9-14-gsoc2021-final.md:25]: Alternatively, something along the lines of looking forward to how they continue to contribute to MDAnalysis. But I didn't manage to come up with something good. Even the "seeing where they take their projects next" does not sound quite right to me. Please edit!!!

**Review by orbeckst — COMMENTED** [review]
thanks @hmacdope , comments inline

File: _posts/2021-9-14-gsoc2021-final.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+Another year, another successful [Google Summer of Code][gsoc]! We had
+[two fantastic projects][gsoc-projects] this year, including the first trials of our MDAKit model for building an ecosystem of analysis modules. Both projects were a resounding success and represent an exciting new direction for MDAnalysis.  None of this would have been possible without our wonderful students: [Estefania Barreto-Ojeda][project-estefania], and [Orion Cohen][project-orion]. 
+
+You can read more about their projects and packages in their blog posts:
+* Estefania: [Curvature for MDA Universes ]({{ site.baseurl }}{% post_url 2021-09-02-gsoc-final-report-estefania%})
+* Orion: [Solvation Analysis]({{ site.baseurl }}{% post_url 2021-09-02-gsoc-final-report-orion%})
+
+
+We immensely enjoyed working with our GSoC students and look forward to seeing where they take their projects next.
```
Comment (lilyminium) [inline _posts/2021-9-14-gsoc2021-final.md:21]:
```suggestion
We have immensely enjoyed working with Estefania and Orion and look forward to seeing where they take their projects next.
```

Feels a little more personal?

File: _posts/2021-9-14-gsoc2021-final.md (hunk 2 above)
Comment (hmacdope) [reply _posts/2021-9-14-gsoc2021-final.md]: How about "how they continue to contribute to the MDAnalysis community and where they take their science next"
It all feels a bit awkward but I couldn't come up with anything better.

**Comment by lilyminium** [comment]
LGTM!

**Comment by IAlibay** [comment]
🤔 I'll be honest - I'm not sure if we should really be calling these the "MDAKit model" - since we've not actually implemented the things that would differentiate MDAKits from normal downstream packages. But I'm happy just ignoring that for now.

**Review by orbeckst — APPROVED** [review]
lgtm, thanks for moving GSOC over the finishing line!

**Pull request merged**